visualization/index.html

```diff
@@ -47,6 +50,14 @@
             $('a[aria-expanded=true]').attr('aria-expanded', 'false');
         });
 
+        $('#showTutorial').on('click', function () {
+            // startIntro()
+            // addHints(this)
+            toggleTutorial(this)
+        });
+
+        $('[data-toggle="tooltip"]').tooltip();
+        $("body").tooltip({ selector: '[data-toggle=tooltip]' });
```

Comment:
A. Can we get rid of the commented lines here?
B. Can we move this entire script tag to a document on load in the JS file used to render the network graph?